.circleci/testnet-deploy-new-chain-for-pr.js

```diff
@@ -33,11 +33,12 @@ if (!targetTag) {
 
 // The namespace 100000 is PR chains teritory
 const prLinkParts = githubPRLink.split('/');
-const prNumber = parseInt(prLinkParts[prLinkParts.length - 1]) + 100000;
+const prNumber = parseInt(prLinkParts[prLinkParts.length - 1]);
+const chainNumber = prNumber + 100000;
 
 const configuration = require(configFilePath);
 
-const chainIndex = configuration.chains.findIndex(chain => chain.Id === prNumber);
+const chainIndex = configuration.chains.findIndex(chain => chain.Id === chainNumber);
```

Comment:
is this the Boyar config? if so, I'd like to see this abstracted in some way that signals that this is Boyar related stuff; if and when Boyar starts to read virtual chain config from a contract, this code will need to change as well and I'd like it to be in one place (here, not in Boyar - but still - not all over the place)

Reply:
If we get to a point where we handle deploying new chains through triggering a smart contract this entire code will vanish or change dramtically. Yes this code alerts the S3 file and re-uploads it adding a new chain to it. How would you like to emphasize it ?

Reply:
just extract this to function with appropriate names